applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.cpp

```diff
@@ -17,13 +17,17 @@
 namespace Kratos
 {
 
+template class TransientThermalElement<2, 2>;
 template class TransientThermalElement<2, 3>;
 template class TransientThermalElement<2, 4>;
+template class TransientThermalElement<2, 5>;
```

Comment:
@avdg81  thank you for the review. 
It is right. We don't have 3D elements with edges of 5 nodes, so we didn't build a such line elements. It is mainly because:
1. We usually used line elements (or more accurately, geometry) for conditions. The condition needs to match with the element order. As we haven't 3D element with a such high order, we never built 3D high order line element.
2. In the case of beams or sheet piles, they need to be connected to soil element. And again they need to match in order.
However, for purly 1D element like thge one in this issue, it is possible to extend and use 3D4N , or 3D5N. It is maybe an issue for future (probably with low priority, as it is not a bloker)